.github/workflows/e2e-deploy.yml

```diff
@@ -19,15 +19,15 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Bump test plugin version number
-        run: sed -i 's/0.0.1/0.0.2/' tests/data/plugins/test-plugin/test-plugin.php
+        run: sed -i 's/0.0.1/0.0.2/' tests/data/wp-content/plugins/test-plugin/test-plugin.php
```

Comment:
The plugin used for e2e tests was moved to reside under a `wp-content` parent dir. The reasoning was to make the file structure more closely resemble a WordPress install, making the plugin reusable for unit testing.